.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Pull Request Checklist
+
+Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:
+
+See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#pull-requests).
+
+- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
+- [ ] Doxygen docs
+- [ ] Unit tests
+- [ ] No unwanted commits/changes
+- [ ] Descriptive title/description
+  - [ ] PR is single purpose
+  - [ ] Related issue listed
+- [ ] Comments in source
+- [ ] No typos
+- [ ] Update changelog
+- [ ] Not work-in-progress
+- [ ] External references or docs updated
+- [ ] Self review of PR done
+- [ ] Any breaking changes?
```

Comment:
still curious about this one..
What if this is a "yes" ?   Should we ask for no-breaking changes instead?

Reply:
Not yet, because we haven't GA'd yet. We should be aware of breaking changes we are making, but the bar is different before we have shipped v1.

Reply:
I'd prefer a shorter reminder list closer to .NET style.  This is a good start and we can tweek over time

Reply:
Agreed, once we start seeing diminishing returns from this (contributors more-or-else have internalized them), we can start shrinking the list.